### ISSUE TITLE
Show the commit hash on the right side when visualizing a file content 

### ISSUE THREAD

**Issue by joseluisq** [go-gitea/gitea#29793]
### Feature Description

In Gitea 1.21.8, it displays the following information when visualizing a file such as its commit and date-time but it seems very constrained.

![image](https://github.com/go-gitea/gitea/assets/1700322/ce76b47b-5b47-4223-941c-ea411aaa47a4)

Source: https://gitea.com/builderdotnet/rpp/src/branch/main/.gitignore

### Screenshots

However, it would be great to show the short commit hash on the right side like the date-time, leaving just the author and the commit message on the left. Similar to how GitHub for example does it. Below is an example:

![image](https://github.com/go-gitea/gitea/assets/1700322/cefd42de-8810-4481-bd51-7ad6835e618f)

**Comment by silverwind** [comment]
I agree that commit hash can be moved to right side. This goes not only for that box, but also for the repo commits table and last commit in the repo.

It also ties in with https://github.com/go-gitea/gitea/issues/29641 which aims to de-emphasize commit hashes to a styling similar to GitHub.

**Comment by silverwind** [comment]
Edited the title: Date is already on right side, so this is only about moving the hash.
